client/profile/user-context-menu.tsx

```diff
@@ -1,34 +1,82 @@
-import React, { useCallback } from 'react'
+import React, { useCallback, useEffect, useRef, useState } from 'react'
```

Comment:
Remove all the changes in this file and let's get this merged, this PR has gone on too long and is too big now for me to really keep coming back to it week after week and trying to put all the context back in my head